<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsCtrls</class>
 <widget class="QWidget" name="SettingsCtrls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log files to:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="logDir">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>53</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Path to where the log files are to be stored.</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browseDir">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>48</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Media/media.qrc">
     <normaloff>:/Mode_Review_96.png</normaloff>:/Mode_Review_96.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum log file size:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="logSize">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>133</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Maximum size of a log file from 50MB to 500MB.</string>
   </property>
   <property name="inputMask">
    <string>0000 M\B</string>
   </property>
   <property name="text">
    <string>500 MB</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Media/media.qrc"/>
 </resources>
 <connections/>
</ui>
